src/components/actionButton/ActionButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+import { ActivityIndicator, Text, TouchableWithoutFeedback, View } from 'react-native';
+import Colors from '@constants/colors';
+import { isString } from '@helpers/helpers';
+import actionButtonStyles from './styles';
+
+const ActionButton = ({ children, onPress, style: customStyle, disabled, loading, addOns }: ActionButtonProps) => {
+  const [styles] = useState(actionButtonStyles());
```

Comment:
why is this in state??

Reply:
![Screenshot 2024-10-29 at 4 45 23 PM](https://github.com/user-attachments/assets/563a5996-1b88-45bf-90c2-ac6724b2affe)
<img width="794" alt="Screenshot 2024-10-29 at 4 50 37 PM" src="https://github.com/user-attachments/assets/e3c93111-546f-455b-afbf-b2ffd14af605">



i have talked with priyanshi regarding this

Reply:
i am fixing this in this pr
removing usage of state